deploy/kubernetes/base/node_windows/node.yaml

```diff
@@ -52,22 +52,28 @@ spec:
               mountPropagation: "None"
             - name: plugin-dir
               mountPath: C:\csi
-            - name: csi-proxy-disk-pipe
+            - name: csi-proxy-disk-v1beta1
               mountPath: \\.\pipe\csi-proxy-disk-v1beta1
-            - name: csi-proxy-volume-pipe
+            - name: csi-proxy-volume-v1beta1
               mountPath: \\.\pipe\csi-proxy-volume-v1beta1
-            - name: csi-proxy-filesystem-pipe
+            - name: csi-proxy-filesystem-v1beta1
               mountPath: \\.\pipe\csi-proxy-filesystem-v1beta1
+            - name: csi-proxy-disk-v1beta2
+              mountPath: \\.\pipe\csi-proxy-disk-v1beta2
       volumes:
-      - name: csi-proxy-disk-pipe
+      - name: csi-proxy-disk-v1beta1
```

Comment:
is v1beta still used?

Reply:
For latest master version of PD driver, v1beta1 is not used. But I am thinking to keep it because this base node yaml file will be used by different versions of pd driver, it seems safer to keep it.

Reply:
Do we have an overlay in the repo that still uses it?

Reply:
Maybe add a comment when this is safe to remove

Reply:
added a comment. I will create another PR to remove v1beta1.